frontend/app/module/polling_stations/page/PollingStationUpdatePage.tsx

```diff
@@ -0,0 +1,39 @@
+import { useNavigate } from "react-router-dom";
+
+import { PollingStationForm } from "app/component/form/polling_station/PollingStationForm";
+
+import { PollingStation, usePollingStationGet } from "@kiesraad/api";
+import { Loader, PageTitle } from "@kiesraad/ui";
+import { useNumericParam } from "@kiesraad/util";
+
+export function PollingStationUpdatePage() {
+  const electionId = useNumericParam("electionId");
+  const pollingStationId = useNumericParam("pollingStationId");
+  const navigate = useNavigate();
+
+  const { requestState } = usePollingStationGet(pollingStationId);
+
+  const onSave = (ps: PollingStation) => {
```

Comment:
```suggestion
  const handleSaved = (ps: PollingStation) => {
```

Reply:
Hope I'm not being difficult, any specific reason for this? (If this is the convention we use thats good enough for me)
onSave is intended as a listener, handleSaved feels like it does something to the saved item.

Reply:
It is mostly about the save vs. saved, because this is not actually saving but will be called after it has been saved, if I understand this correctly. 
The `handle` vs. `on` is what I thought a convention indeed.

Reply:
handleSaved